plugin.json

```diff
@@ -2,7 +2,7 @@
     "id": "mattermost-plugin-azure-devops",
     "name": "Mattermost Azure DevOps plugin",
     "description": "This plugin provides services of Azure DevOps such as Boards and Repos",
-    "icon_path": "assets/azurebot.svg",
+    "icon_path": "public/azurebot.svg",
```

Comment:
we should move the `assets` folder inside `public`
the path would be `public/assets/abc.svg`